django_webapps/scripts/db_population/parsers/student_parsers.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from api.models.auxil.constants import STUDENT_AGE_RANGES
+from api.models.choices.status.project import StudentProjectStatus
+from api.models.choices.status.situational import StudentSituationalStatus
+
+RAW_PROJECT_STATUS_TO_PARSED: dict[
+    str, tuple[StudentSituationalStatus | None, StudentProjectStatus]
+] = {
+    "группа расформирована": (None, StudentProjectStatus.FINISHED),
+    "не ходит на занятия": (StudentSituationalStatus.NOT_ATTENDING, StudentProjectStatus.STUDYING),
+    "идут занятия": (None, StudentProjectStatus.STUDYING),
+    "нет ответа": (StudentSituationalStatus.NO_RESPONSE, StudentProjectStatus.NO_GROUP_YET),
+    "перестал отвечать": (StudentSituationalStatus.NO_RESPONSE, StudentProjectStatus.NO_GROUP_YET),
+    "вышел на связь, ожидает старта занятий": (
+        StudentSituationalStatus.AWAITING_START,
+        StudentProjectStatus.STUDYING,
+    ),
+    "уточнение расписания": (None, StudentProjectStatus.NO_GROUP_YET),
+    "жду ответа": (StudentSituationalStatus.NO_RESPONSE, StudentProjectStatus.NO_GROUP_YET),
```

Comment:
```suggestion
    "жду ответа": (None, StudentProjectStatus.NO_GROUP_YET),
```
we should set `.NO_RESPONSE` after a while. "жду ответа" is a neutral status (like, we wrote to the student but not enough time has passed to think that they're ignoring us).